ircstream/ircserver.py

```diff
@@ -782,6 +782,7 @@ def __init__(self, config: configparser.SectionProxy) -> None:
             "channels": Gauge("ircstream_channels", "Number of IRC channels", registry=registry),
             "messages": Counter("ircstream_messages", "Count of RC messages broadcasted", registry=registry),
             "errors": Counter("ircstream_errors", "Count of errors and exceptions", ["type"], registry=registry),
+            "client_per_channel": Gauge("ircstream_clients_per_channel", "Number of IRC clients", ["channel"], registry=registry),
```

Comment:
The label needs to be different than the "clients" metric.

Reply:
Sorry, I was coding C# at the same time :-)